### PR TITLE
Balls bot pythonvariant :grin:

### PR DESCRIPTION
**Pull request by WeDoABitOfTrolling**

### CODE REVIEW

File: BallsDiscordBot.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import discord as Ballspy
+from Ballspy.ext import commands as BallsCommands
+
+Balls = BallsCommands.bot(command_prefix="sex")
```
Comment (ballsballsballsballsballsballsballsball) [inline BallsDiscordBot.py:4]:
```suggestion
Balls = BallsCommands.bot(command_prefix="balls”)
```
hhaha look what i learned how to do

Comment (ballsballsballsballsballsballsballsball) [inline BallsDiscordBot.py:4]:
```suggestion
Balls = BallsCommands.bot(command_prefix="balls")
```

**Review by ballsballsballsballsballsballsballsball — CHANGES_REQUESTED** [review]
WAIT

**Comment by WeDoABitOfTrolling** [comment]
sus

**Comment by ceilingfans** [comment]
python :dead:

**Comment by WeDoABitOfTrolling** [comment]
nop

**Pull request merged**